src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -1092,6 +1109,15 @@ export default defineComponent({
       emit('ended')
     }
 
+    function handleCanPlay() {
+      // PiP can only be activated once the video's readState and video track are populated
+      if (startInPip && ui.getControls().isPiPAllowed() && process.env.IS_ELECTRON) {
```

Comment:
PiP is not possible in audio-only mode, I'm not sure if that is already caught by the isPipAllowed() method but if it isn't we should check for it.
```suggestion
      if (startInPip && props.format !== 'audio' && ui.getControls().isPiPAllowed() && process.env.IS_ELECTRON) {
```